modules/blog/post-body/post-body.tsx

```diff
@@ -175,6 +179,10 @@ const renderOptions = (
                   height={entry.image.height}
                   src={entry.image.url}
                   width={entry.image.width}
+                  style={{
```

Comment:
Optional: put this in a cost and call it where it's needed.

Reply:
Agreed. It's used very often now.